src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,31 @@
+package racingcar;
+
+import java.util.Random;
+
+public class Car {
+
+    private final String name;
+    private int position;
+
+    public Car(final String name, int position) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException();
+        }
+        this.name = name;
+        this.position = position;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void move(final int condition) {
+        move(() -> condition >= 4);
+    }
+
+    public void move(final MovingStrategy movingStrategy) {
+        if (movingStrategy.movable()) {
+            position++;
+        }
+    }
+}
```

Comment:
파일의 마지막 라인에 개행을 하는것은 어떨까요?


```suggestion
    }
}

```

래퍼런스 : https://jojoldu.tistory.com/673